data/hai ships.txt

```diff
@@ -651,3 +651,56 @@ ship "Water Bug" "Water Bug (Jump)"
 		`"Benga" Atomic Thruster`
 		`"Benga" Atomic Steering`
 		"Jump Drive"
+
+ship "Centipede"
```

Comment:
Ships should be placed in their ship.txt files alphabetically; you've placed each of these at the end of the file.